onnxruntime/python/tools/transformers/bert_perf_test.py

```diff
@@ -52,9 +53,11 @@ class ModelSetting:
     opt_level: int
 
 
-def create_session(model_path, use_gpu, provider, intra_op_num_threads, graph_optimization_level=None):
+def create_session(model_path, use_gpu, provider, intra_op_num_threads, log_severity, graph_optimization_level=None):
```

Comment:
For backward compatible, please move log_severity=2 to the last parameter: Just in case this function is used in other py file.